src/rpc/net.cpp

```diff
@@ -872,8 +872,8 @@ static RPCHelpMan getnodeaddresses()
 
     for (const CAddress& addr : vAddr) {
         UniValue obj(UniValue::VOBJ);
-        obj.pushKV("time", (int)addr.nTime);
-        obj.pushKV("services", (uint64_t)addr.nServices);
+        obj.pushKV("time", static_cast<int>(addr.nTime));
```

Comment:
Hmm, I think we generally use C-style casts for primitive types (as they are syntactically simpler, and for those types they're all identical in semantics anyway). No problem using these in new code you write yourself of course, but mostly a matter of personal taste.

Reply:
Ideally there'd be no casts here, but I suspect that will require some changes in univalue

Reply:
Dropped the casts commit. Definitely an optional change.

(https://github.com/bitcoin/bitcoin/pull/20965#issuecomment-770375623 summarizes why I tended to use named casts even for primitives.)